sdk/src/manifest.rs

```diff
@@ -1023,11 +1023,30 @@ impl Manifest {
 
     /// Embed a signed manifest into a stream using a supplied signer.
     /// returns the bytes of the new asset
+    /// We plan to deprecate this, please use embed_to_stream instead
```

Comment:
Is it worth adding the `#[deprecated]` tag now?

Reply:
I updated this to be deprecated, which, Rust being Rust, then forced me to update any other calls we had to it in our SDK so that I didn't get warnings.